pkg/container/container.go

```diff
@@ -62,6 +71,21 @@ type Container interface {
 	// SelinuxLabel returns the container's SelinuxLabel
 	// it takes the sandbox's label, which it falls back upon
 	SelinuxLabel(string) ([]string, error)
+
+	// spec functions
+
+	// returns the spec
+	GetSpec() *generate.Generator
```

Comment:
idiomatic go says to drop `Get` from getters, so this should just be `Spec()`

Reply:
fixed!